libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -1115,6 +1115,8 @@ void AP_GPS::update_primary(void)
         for (uint8_t i=1; i<GPS_MAX_RECEIVERS; i++) {
             // choose GPS with highest state or higher number of satellites
             if ((state[i].status > state[primary_instance].status) ||
+                ((now - state[primary_instance].last_gps_time_ms) > 400 &&
```

Comment:
randy has asked to break out into bool vars

Reply:
I'm fine with it broken out or not, but if we can have a define or a const for the time that would be nice.

Reply:
done